source/common/http/http1/parser.h

```diff
@@ -96,7 +96,7 @@ class ParserCallbacks {
   /**
    * Called when accepting a chunk header.
    */
-  virtual void onChunkHeader(bool) PURE;
+  virtual void onChunkHeader(int content_length) PURE;
```

Comment:
We should revert onChunkHeader (and all other changes in other files related to this e.g. legacy_parser_impl, http1/codec_impl) since we decided to count header bytes.